crates/assists/src/handlers/replace_if_let_with_match.rs

```diff
@@ -7,10 +7,8 @@ use syntax::{
     AstNode,
 };
 
-use crate::{
-    utils::{unwrap_trivial_block, TryEnum},
-    AssistContext, AssistId, AssistKind, Assists,
-};
+use crate::{utils::unwrap_trivial_block, AssistContext, AssistId, AssistKind, Assists};
+use ide_db::ty_filter::TryEnum;
```

Comment:
Very minor and is not required to be fixed, but we put `crate::` and `other_crate::` deps in different groups: https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#order-of-imports